src/xo-app/jobs/overview/index.js

```diff
@@ -0,0 +1,426 @@
+import ActionButton from 'action-button'
+import ActionRowButton from 'action-row-button'
+import ActionToggle from 'action-toggle'
+import Icon from 'icon'
+import React, { Component } from 'react'
+import _, { FormattedDuration } from 'messages'
+import ceil from 'lodash/ceil'
+import classnames from 'classnames'
+import filter from 'lodash/filter'
+import forEach from 'lodash/forEach'
+import map from 'lodash/map'
+import orderBy from 'lodash/orderBy'
+import { FormattedDate } from 'react-intl'
+import { ButtonGroup, Pagination } from 'react-bootstrap-4/lib'
+import { Link } from 'react-router'
+import { confirm } from 'modal'
+import { connectStore } from 'utils'
+import { createGetObject, createPager } from 'selectors'
+
+import {
+  Card,
+  CardHeader,
+  CardBlock
+} from 'card'
+
+import {
+  deleteJobsLog,
+  deleteSchedule,
+  disableSchedule,
+  enableSchedule,
+  runJob,
+  subscribeJobs,
+  subscribeJobsLogs,
+  subscribeSchedules,
+  subscribeScheduleTable
+} from 'xo'
+
+import { getJobValues } from '../helpers'
+
+// ===================================================================
+
+const jobKeyToLabel = {
+  genericTask: _('customJob')
+}
+
+// ===================================================================
+
+const LOGS_PER_PAGE = 10
+
+@connectStore(() => {
+  const object = createGetObject((_, props) => props.value)
+  return (state, props) => ({object: object(state, props)})
```

Comment:
Since a few days ago, you can returns directly an object containing the selectors and the wrapping function is created automatically for you:

```js
return (state, props) => ({
  object: getObject(state, props),
  pool: getPool(state, props)
})

// … becomes …
return {
  object: getObject,
  pool: getPool
}
```

Feel free to use the pattern you prefer.